WMS ID#7321  Get Started with MySQL-HeatWave and Feel the Difference.

jitender-git opened the pull request:
The workshop was already approved and merged into master library however while doing QA with WMS team, they suggested some minor changes (like adding #Acknowledgement). Hence request you to please approve so that it can be merged and published in production.

anooshapilli commented on data-management-library/mysql/mysql-heatwave/4execute-sql-queries/execute-sql-queries.md:
```diff
@@ -18,7 +18,7 @@ This lab assumes you have:
 * All previous labs successfully completed
 
-## **TASK 4**: Execute SQL Queries to See the difference
+## **Task 4**: Execute SQL Query to see the Performance Differences
```
@jitender-git , Please rename Task 4 to Task 1

anooshapilli commented on data-management-library/mysql/mysql-heatwave/3upload-data-into-heatwave/upload-data-into-heatwave.md:
```diff
@@ -19,9 +19,9 @@ This lab assumes you have:
 * All previous labs successfully completed
 
-## **TASK 3**: Upload data into MySQL HeatWave Cluster Engine
+## **Task 3**: Upload data into MySQL HeatWave Cluster Engine
```
@jitender-git , Please rename Task 3 to Task 1

anooshapilli commented on data-management-library/mysql/mysql-heatwave/2enable-heatwave-cluster/enable-heatwave-cluster.md:
```diff
@@ -18,9 +18,9 @@ This lab assumes you have:
 * All previous labs successfully completed
 
-## **TASK 2**: Enable HeatWave Cluster to MySQL Database System
+## **Task 2**: Enable HeatWave Cluster in MySQL Database System
```
@jitender-git , Please rename Task 2 to Task 1

anooshapilli reviewed (commented):
@jitender-git , Please remove the numbers before of the folder name. For example, Remove 1 before the folder 1create-mysql-database-system in the path - data-management-library/mysql/mysql-heatwave/1create-mysql-database-system/create-mysql-database-system.md

jitender-git commented:
Hi @anooshapilli,

As suggested, I have modified the folder name (removed numbering) and changed Task. Please review and approve the changes.

Thanks,

anooshapilli commented:
@jitender-git Please unbold the Tasks. i.e Task 1

jitender-git commented:
Hi, done the changes. Please review and approve.
